.travis.yml

```diff
@@ -23,18 +33,32 @@ env:
 before_script:
 - mkdir -p $HOME/.sbt/launchers/0.13.8/
 - curl -L -o $HOME/.sbt/launchers/0.13.8/sbt-launch.jar http://dl.bintray.com/typesafe/ivy-releases/org.scala-sbt/sbt-launch/0.13.8/sbt-launch.jar
+- ./dist/create_bridge_config.sh
 script:
-- sbt clean test
+- sbt clean test dist
 branches:
   only:
   - develop
   - uat
   - prod
 deploy:
-  provider: heroku
-  api_key:
-    secure: dsrD0lN7luCXQCI+AK4wBZEWpxv/ji2xbbSMWWo3snBYb8CbuzqmH3KHSWStd9kSTKojWzatGFfIoIGGj6qoyG7gAk4E/PR1tb2Gelz+t6yWQUcRQclLHNd/WkozHLb5Z9Bg7N4HLcyupl7OMICixAaQ92+o2M+xhHq3/r2aOFs=
-  app:
-    develop: bridge-develop
-    uat: bridge-uat
-    prod: bridge-prod
+  matrix:
+  - provider: heroku
+    api_key:
+      secure: dsrD0lN7luCXQCI+AK4wBZEWpxv/ji2xbbSMWWo3snBYb8CbuzqmH3KHSWStd9kSTKojWzatGFfIoIGGj6qoyG7gAk4E/PR1tb2Gelz+t6yWQUcRQclLHNd/WkozHLb5Z9Bg7N4HLcyupl7OMICixAaQ92+o2M+xhHq3/r2aOFs=
+    app:
+      develop: bridge-develop
+      uat: bridge-uat
+      prod: bridge-prod
+  - provider: elasticbeanstalk
+    skip_cleanup: true
+    bucket_name: org-sagebridge-deployment-dev
+    zip_file: target/universal/bridgepf-0.1-SNAPSHOT.zip
+    access_key_id: AKIAIG6T67NMROEIMC7A
+    secret_access_key:
+      secure: L3Z9f5KvDbMr6HvDXz9PfEQBdRTF8PUkoqGe2JsCOFADSvwPUWlnLOjrEtCVpSyf88b0F3TUQp5AKd8qS7waHwiYBRpXWV5ABM8bF7SOSCurt41C1iuyJ+oiN6czqmRkW3ijMTtfFPxf+n86XIArp/26l8LxIDaaNwiZbU+X190=
+    region: us-east-1
+    app: BridgePF-heroku-dev
```

Comment:
For other services, our current pattern is we have an app for the service and an env for each stage. (See for example BridgeEX.) Is it better to have a separate app for each stage? It seems like it would be better to manage one app instead of 3.